### DISCUSSION TITLE
which data provider for commercial applications?

### DISCUSSION THREAD

**Discussion by tfriedel** [opengeos/segment-geospatial#366] [Q&A] [answered]
Let's say I want to use sam for segmenting some very high resolution satellite imagery and sell this processed data, what are my options for providers that allow me to do this?
I checked for example google maps or esri, but their term don't allow this. There are some providers that sell you data at eye-watering prices, like Airbus for example. They are not a realistic option if you want to create crop maps for example.
I know Sentinel data is freely available, but it's not very high resolution.

**Comment by giswqs** [comment, marked as answer]
USDA NAIP imagery, available in the U.S. only 

https://naip-usdaonline.hub.arcgis.com/

**Comment by tfriedel** [comment]
thanks, that's great but like you said US-only. I need especially India.
